LBH.AdultSocialCare.Data/Constants/Enums/PayrunType.cs

```diff
@@ -9,13 +9,13 @@ public enum PayrunType
         [Description("Residential Recurring")]
         ResidentialRecurring = 1,
 
-        [Description("Direct Payments")]
-        DirectPayments = 2,
+        /*[Description("Direct Payments")]
```

Comment:
Don't angry me :) Can you reseed value? remove uncomment, set ResidentialReleasedHolds = 2, that's all

Reply:
Updated